x/farming/keeper/proposal_handler_test.go

```diff
@@ -189,7 +191,9 @@ func (suite *KeeperTestSuite) TestAddPublicPlanProposal() {
 				err := keeper.HandlePublicPlanProposal(suite.ctx, suite.keeper, proposal)
 				suite.Require().NoError(err)
 
-				_, found := suite.keeper.GetPlan(suite.ctx, uint64(1))
+				plan, found := suite.keeper.GetPlan(suite.ctx, uint64(1))
+				fmt.Println(plan)
+				// TODO: need to check each field same as expected
```

Comment:
I'll add validation to check each field with other TODO lists in the other PR.

Reply:
Yes Please, there was the same bug mentioned above, and which was already resolved in the PR commit.
Prior to modification, there was a bug in the PublicPlan Creation where TerminationAddress was not used in AddRequestProposal and FarmingPoolAddress was used.